src/style.css

```
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  width: 100px;
+  height: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.stars > .stars_star {
+  width: 13px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url("images/star.svg");
+}
```

No need to increase specificity.

```suggestion
.stars_star {
  width: 13px;
  background-repeat: no-repeat;
  background-position: center;
  background-image: url("images/star.svg");
}
```